indico/MaKaC/common/utils.py

```diff
@@ -414,13 +406,25 @@ def lockFile(cls, f, lockType):
 
 
 def getProtectionText(target):
-    if target.hasAnyProtection():
-        if target.isItselfProtected():
-            return "protected_own", None
-        elif target.hasProtectedOwner():
-            return "protected_parent", None
-        elif target.getDomainList() != []:
-            return "domain", list(x.getName() for x in target.getDomainList())
+    # XXX: the only relevant return values are domain / not falsy
+    from MaKaC.conference import Conference
+    if isinstance(target, Conference):
+        event = target.as_event
+        if not event.is_protected:
+            return '', None
+        networks = [x.name for x in event.get_access_list() if x.is_network]
+        if networks:
+            return 'domain', networks
         else:
-            return getProtectionText(target.getOwner())
-    return "", None
+            return 'protected', None
+    else:
```

Comment:
only legacy categories for now, so this will go away when we move the category display pages to jinja